<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>decrypt_menu</class>
 <widget class="QMainWindow" name="decrypt_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hides - El ocultador de archivos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="unzip_file_search">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/zip-file-search.png</normaloff>../Icons/zip-file-search.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="unzip_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Archivo secreto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_dir_search">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/folder-search.png</normaloff>../Icons/folder-search.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_dir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Directorio de destino</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/terminal.png</normaloff>../Icons/terminal.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
